src/scripts/components/layer/wv.layer.list.js

```diff
@@ -99,21 +101,33 @@ export default class LayerList extends React.Component {
     </CellMeasurer>
     );
   }
+  getLoader() {
+    if(this.state.metadataLoaded) {
+      return;
+    }
+    return (
+      <div className="loader">Loading Layer descriptions...</div>
+    );
+  }
   render() {
     var { height, width } = this.state;
+    var loader = this.getLoader();
     return(
-      <List
-        deferredMeasurementCache={this._cache}
-        id="flat-layer-list"
-        width={width}
-        height={height}
-        overscanRowCount={5}
-        ref={ref=>this._setListRef(ref)}
-        rowCount={this.state.layerFilter.length}
-        rowHeight={this._cache.rowHeight}
-        scrollToAlignment="auto"
-        rowRenderer={row=>this._rowRenderer(row)}
-      />
+      <div>
+        {loader}
```

Comment:
Rather than making this a separate function and adding a wrapping div, what do you think of doing something like this?

```
render() {
  var { height, width, isMetadataLoaded } = this.state;
  return(
    {isMetadataLoaded
      ?<List>...</List>
      :<div className"loader">Loading layer descriptions</div>
    }
  );
}
```

Or the reverse;

```
render() {
  var { height, width, isMetadataLoaded } = this.state;
  return(
    {!isMetadataLoaded
      ?<div className"loader">Loading layer descriptions</div>
      :<List>...</List>
    }
  );
}
```